[Docs] Remove setting of `pio.templates.default` on import (#631)
mckinsey/vizro: vizro-core/docs/pages/user-guides/themes.md

# How to use themes

This guide shows you how to use themes. Themes are pre-designed collections of stylings that are applied to entire charts and dashboards.

> The themes provided by Vizro are infused with our design best practices that make charts and dashboards look visually consistent and professional.

Reviewer: 😆 that's a nice catch!

## Themes in dashboards
The [`Dashboard`][vizro.models.Dashboard] model accepts an optional `theme` argument, where you can choose between

> a `vizro_dark` and a `vizro_light` theme. If not specified then `theme` defaults to `vizro_dark`. The theme will be applied to the entire dashboard and its charts/components when a user first loads your dashboard. Regardless of the theme applied on first load, users can always switch between light and dark themes via the toggle button in the upper-right corner of the dashboard.

Reviewer: ```suggestion
a `vizro_dark` and a `vizro_light` theme. If not specified then `theme` defaults to `vizro_dark`. The theme is applied to the entire dashboard and its charts/components when a user first loads your dashboard. Regardless of the theme applied on first load, users can always switch between light and dark themes via the toggle button in the upper-right corner of the dashboard.
```
